src/components/Content/ProductList/productList.jsx

```diff
@@ -0,0 +1,44 @@
+import SideBar from "./SideBar/sideBar";
+import Grid from "../Grid/grid";
+
+import './productList.css'; 
+import { useState } from "react";
+
+import products from '../../../mocks/es-mx/products.json';
+
+const ProductList = (props) => {
```

Comment:
aquí intenta hacer destructure para los props, para saber fácilmente cuales son los que te están llegando. 
```suggestion
const ProductList = ( { categories } ) => {
```